README.md

```diff
@@ -132,9 +132,19 @@ docker build --tag=$USER/gitlab .
 # Quick Start
 
 The quickest way to get started is using [docker-compose](https://docs.docker.com/compose/).
-
 ```bash
 wget https://raw.githubusercontent.com/sameersbn/docker-gitlab/master/docker-compose.yml
+```
+
+You just have to create your own `docker-compose.yml` file.
+Copy from our sample file
+```bash
+cp docker-compose.yml.dist docker-compose.yml
+```
+You can edit your `docker-compose.yml` file to change our default values.
+And now, you just have to run
+
+```bash
 docker-compose up
```

Comment:
I think the instructions should be:

```bash
wget https://raw.githubusercontent.com/sameersbn/docker-gitlab/master/docker-compose.yml.dist -O docker-compose.yml
docker-compose up
```

Its a quickstart guide, copy-paste commands are preferred here :smile: